examples/e16_sqlite_database/src/main.rs

```diff
@@ -1,6 +1,8 @@
 // It is recommended that you read the README file, it is very important to this example.
 // This example will help us to use a sqlite database with our bot.
 
+use std::fmt::Write as _;
```

Comment:
```suggestion
use std::fmt::Write;
```

Ok, last one for real

Reply:
Why? The Write trait is not referenced anywhere, we only need it in scope for `writeln!` to work

Reply:
I think these have the same effect, right? Technically, the first one means you can't name the trait explicitly, but I don't think it's a problem, plus it looks a bit messy.

Reply:
I think this is entirely subjective

To make sure, I asked on the community server (https://discord.com/channels/273534239310479360/818964227783262209/1025179266209562654) and they could find no strong objective reason for one or the other. The tendency was towards `as _` to avoid conflicts if the user copy-pastes the example, adds file IO and imports `std::io::Write`.

Reply:
Ok, makes sense. I won't belabor the point further.